runelite-client/src/main/java/net/runelite/client/util/ImageUtil.java

```diff
@@ -361,7 +361,7 @@ public static BufferedImage getResourceStreamFromClass(final Class c, final Stri
 	 * @param color The color with which to fill pixels.
 	 * @return      The given image with all non-transparent pixels set to the given color.
 	 */
-	static BufferedImage fillImage(final BufferedImage image, final Color color)
+	public static BufferedImage fillImage(final BufferedImage image, final Color color)
```

Comment:
The only places I saw these functions called were in the tests, so I didn't see any reason why they couldn't be public.

Reply:
(a) These aren't called outside of ImageUtil, thus don't need to be exposed. (b) Why the heck are these changes part of a completely unrelated PR?